modules/managing-cluster-capabilites.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// *post_installation_configuration/cluster-capabilities.adoc
+
+[id="managing_the_cluster_capabilities_{context}"]
+= Managing the cluster capabilities
+
+As a cluster administrator, you can manage the capabilities by using OpenShift CLI (oc).
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have installed the jq package.
+
+.Procedure
+
+* To manage the `ClusterVersion` resource, run the following command:
++
+[source,terminal]
+----
+$ oc patch clusterversion version --type json -p '[{"op": "replace", "path": "/spec/capabilities", "value": {"baselineCapabilitySet": "vCurrent"}}]' <1>
```

Comment:
The command would override the `additionalEnabledCapabilities` if it's present.

```
oc patch clusterversion version --type merge -p '{"spec":{"capabilities":{"baselineCapabilitySet":"vCurrent"}}}'
```
The command ^ could update or set baselineCapabilitySet w/o dropping `additionalEnabledCapabilities`

Reply:
fixed.